app/src/main/java/org/standardnotes/notes/SettingsActivity.kt

```diff
@@ -35,6 +38,24 @@ class SettingsActivity : BaseActivity() {
         return super.onOptionsItemSelected(item)
     }
 
+    override fun onDestroy() {
+        super.onDestroy()
+        // TODO does this make sense here?
+        ExportUtil.clearExports(this)
```

Comment:
I think best not to clear the dir unless the user is either logging out or they are doing a new export (and clear before the file writes can be done)